prost-build/src/lib.rs

```diff
@@ -578,4 +578,23 @@ mod tests {
         let actual = String::from_utf8(buf).unwrap();
         assert_eq_fixture_contents!("src/fixtures/write_includes/_.includes.rs", actual);
     }
+
+    #[test]
+    fn test_generate_deprecated() {
```

Comment:
This test is better than `deprecated_field`, as the existing test doesn't verify the existence of the deprecated attributes. Please remove the existing test and this test and related files to `deprecated_field`.